project-novica/demo/src/main/java/com/example/demo/controller/QuestionController.java

```diff
@@ -33,4 +34,10 @@
             return ResponseEntity.badRequest().body(res);
         }
     }
+
+    @GetMapping("/answer")
+    public String AnswerPage()
```

Comment:
This is not anymore logic for question. Create AnswerController and place this endpoint there.